web/public/locales/no-NB.json

```diff
@@ -705,7 +752,7 @@
       "zoneName": "Guyana"
     },
     "HK": {
-      "zoneName": "Hong Kong"
+      "zoneName": "Hong-Kong"
```

Comment:
```suggestion
      "zoneName": "Hongkong"
```

At least according to Wikipedia 😅

Reply:
Seems like this was not changed, did you perhaps forget to push it or was the dashed version actually correct? :)